pkg/v1/remote/transport/error.go

```diff
@@ -35,7 +39,7 @@ var _ error = (*Error)(nil)
 func (e *Error) Error() string {
 	switch len(e.Errors) {
 	case 0:
-		return "<empty transport.Error response>"
+		return fmt.Sprintf("unsupported status code %d; body: %q", e.StatusCode, e.rawBody)
```

Comment:
Not super important, but I would guess `%q` is going to look bad pretty often for multi-line bodies. It might also be nice to just omit the body if it's empty.

Reply:
Used %s and skipped body if empty.